doc/modules/preprocessing.rst

```diff
@@ -219,13 +219,16 @@ of the data is likely to not work very well. In these cases, you can use
 :class:`RobustScaler` as a drop-in replacement instead. It uses
 more robust estimates for the center and range of your data.
 
-
-.. topic:: References:
+|details-start|
+**References**
+|details-split|
 
   Further discussion on the importance of centering and scaling data is
   available on this FAQ: `Should I normalize/standardize/rescale the data?
   <http://www.faqs.org/faqs/ai-faq/neural-nets/part2/section-16.html>`_
 
+|details-end|
```

Comment:
Can you please move this dropdown right after the `Scaling vs. Whitening` topic? Just an aesthetic tweak.

Optionally, you can also hide said topic inside a dropdown.